### DISCUSSION TITLE
GPyTorch Regression Tutorial gives "you must train on training inputs!"

### DISCUSSION THREAD

**Discussion by Atulsingh92** [cornellius-gp/gpytorch#2273] [Q&A]
Hello, 

Tried the basic "GPyTorch Regression Tutorial" for v1.6.0, given [here](https://docs.gpytorch.ai/en/v1.6.0/examples/01_Exact_GPs/Simple_GP_Regression.html),
get an error at the line
```

 f_preds = model(test_x)

  File ~\Anaconda3\envs\gpytorch_env\lib\site-packages\gpytorch\models\exact_gp.py:256 in __call__
    raise RuntimeError("You must train on the training inputs!")

RuntimeError: You must train on the training inputs!
```
Could someone help?

**Comment by gpleiss** [comment]
Before calling `f_preds = model(test_x)`, call `model.eval()`.
